docs/start/install.mdx

```diff
@@ -360,6 +360,17 @@ release had some issue with the sdist package publishing, and a post-release
 0.25.2.1 only fixed the packaging issue for the release.
 </details>
 
+<details>
+<summary id="qpy-support">
+QPY support
+</summary>
```

Comment:
I would move this up to be adjacent to the upgrade strategy section before the pre and post-release sections. Realistically my old repeated complaint is that we need more hierarchy to the headings in this section of the docs is really showing here, because this will look pretty awkward on the rendered docs.

Reply:
@javabster @Eric-Arellano what do you think?

Reply:
moving it next to upgrade strategy makes sense to me. Also we will soon begin working on [this issue](https://github.com/Qiskit/documentation/issues/605) to move some of the more open-source focused information into a dedicated section, instead of trying to squeeze it all into the install page, so hopefully this should help ease the header hierarchy issues

Reply:
This means we should merge this as suggested in this PR and move it with #605 ?

Reply:
it means we can move it next to the upgrade strategy for this PR. I just mentioned #605 in response to matthews comment about not liking the header hierarchy in general

Reply:
done and squashed. Thanks!